src/main/java/org/mskcc/cbio/oncokb/web/rest/AccountResource.java

```diff
@@ -322,39 +368,71 @@ public void deleteToken(@RequestBody Token token) throws AuthenticationException
     }
 
     /**
-     * {@code POST   /account/reset-password/init} : Send an email to reset the password of the user.
+     * {@code POST   /account/reset-password/init} : Send an email to reset the
+     * password of the user.
      *
      * @param mail the mail of the user.
+     * @throws Exception
      */
     @PostMapping(path = "/account/reset-password/init")
-    public void requestPasswordReset(@RequestBody String mail) {
-        Optional<User> user = userService.getUserWithAuthoritiesByEmailIgnoreCase(mail);
-        if (user.isPresent()) {
-            Optional<User> updatedUser = userService.requestPasswordReset(user.get().getLogin());
-            if (updatedUser.isPresent()) {
-                mailService.sendPasswordResetMail(userMapper.userToUserDTO(updatedUser.get()));
+    public void requestPasswordReset(@RequestBody String mail, HttpServletRequest request) throws Exception {
+        try {
+            ResponseEntity<String> rs = CreateAssessment.createAssessment(request);
+            // if (rs.getStatusCode() == HttpStatus.OK) {
```

Comment:
This is temporarily commented out so that a poor score will not block the user from resetting their password. We can un-comment this after the trial period